handlers/retest/ci.go

```diff
@@ -0,0 +1,215 @@
+package retest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	. "github.com/Huawei-PaaS/ci-bot/handlers/types"
+
+	"github.com/golang/glog"
+)
+
+const (
+	TravisCIEndPoint = "https://api.travis-ci.org" //travis-ci endpoint
+	ContentTypeJSON  = "application/json"          //content-type
+	TravisAPIVersion = "Travis-API-Version"        //API version, a Mandatory header field for Travis-CI V3 APIs call
+)
+
+type TravisJobType string
+
+//Travis Build Job Names
+const (
+	Verify      TravisJobType = "verify"   //Travis Build Job Name
```

Comment:
//Travis Build Job Name 

is not a meaningful comment message. Remove them or give a better comment.